pkg/config/v2/loadbalancer.go

```diff
@@ -22,7 +22,8 @@ import (
 )
 
 type LeastRequestLbConfig struct {
-	ChoiceCount uint32
+	ChoiceCount       uint32
+	ActiveRequestBias float64 `json:"active_request_bias,omitempty"`
```

Comment:
we'd better write some comments for these configurations, otherwise, LGTM.

Reply:
Resolved